src/theory/arith/theory_arith_private.h

```diff
@@ -828,11 +828,16 @@ class TheoryArithPrivate {
 
   Statistics d_statistics;
 
-  enum ArithSkolemId
+  enum class ArithSkolemId
```

Comment:
Why is the "class" keyword here?

Reply:
[class enums](https://en.cppreference.com/w/cpp/language/enum) make the constants scoped, i.e. you _have_ to refer to them as `ArithSkolemId::*` (so you don't need to prefix your constants). At the same time it disallows implicit conversions, so you can't accidentally compare an `ArithSkolemId` to an integer or another `enum`. The [C++ Core Guidlines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-class) recommend the use of class enums over "plain" enums for this reason.